--- a/vendor/circle-stdlib/libs/circle/include/circle/types.h
+++ b/vendor/circle-stdlib/libs/circle/include/circle/types.h
@@ -21,14 +21,15 @@
 #define _circle_types_h
 
 #include <assert.h>
+#include <stdint.h>
 
 typedef unsigned char		u8;
 typedef unsigned short		u16;
-typedef unsigned int		u32;
+typedef uint32_t			u32;
 
 typedef signed char		s8;
 typedef signed short		s16;
-typedef signed int		s32;
+typedef int32_t				s32;
 
 #if AARCH == 32
 typedef unsigned long long	u64;
